radiospectra/spectrogram2/sources.py

```diff
@@ -93,31 +145,58 @@ def observatory_location(self):
         return EarthLocation(lat=lat, lon=lon, height=height)
 
     @classmethod
-    def is_datasource_for(cls, *, meta, data, **kwargs):
+    def is_datasource_for(cls, data, meta, **kwargs):
         return meta['instrument'] == 'e-CALLISTO' or meta['detector'] == 'e-CALLISTO'
 
 
-class EOVSASpectrogram(BaseSpectrogram):
+class EOVSASpectrogram(GenericSpectrogram):
     """
     Extend Owen Valley Array (EOVSA) Spectrogram
+
+    Examples
+    --------
+    >>> import radiospectra.net
+    >>> from sunpy.net import Fido, attrs as a
+    >>> from radiospectra.spectrogram2 import Spectrogram
+    >>> query = Fido.search(a.Time('2021/05/07 00:00', '2021/05/07 23:00'), a.Instrument.eovsa)  #doctest: +REMOTE_DATA
+    >>> downloaded = Fido.fetch(query[0][0])  #doctest: +REMOTE_DATA
+    >>> spec = Spectrogram(downloaded[0])  #doctest: +REMOTE_DATA
+    >>> spec  #doctest: +REMOTE_DATA
+    <EOVSASpectrogram OWENS VALLEY, EOVSA, EOVSA 1105371.117591858 kHz - 17979686.737060547 kHz, 2021-05-07T13:48:20.999 to 2021-05-08T01:50:59.999>
+    >>> spec.plot()  #doctest: +REMOTE_DATA
```

Comment:
Do these show up in the final doc build?

Reply:
The plot does not but the rest does.

Reply:
Fixed

Reply:
In sunpy we skip them, I am unsure why.